capstone/heroku.md

```diff
@@ -10,42 +10,88 @@ $ brew tap heroku/brew && brew install heroku
 $ heroku --version
 ```
 
-3. Now we can login to Heroku from the command line. Enter your email and password for your group Heroku account.
+3. First we'll use Heroku's browser authenticated login in via the command line
```

Comment:
To maintain sentence structure, start each sentence with a verb. 
`Use Heroku's browser authenticated login in via the command line`